pkgs/development/python-modules/dendropy/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  name    = "${pname}-${version}";
```

Comment:
no `name`

Reply:
I tried omitting name but that resulted in a failure when I tried locally

Reply:
There shouldn't be any. Did you rebase on latest master?
Btw, `name` is optional only for `buildPythonPackage` and `buildPythonApplication`.